docs/par2cron_check-config.md

```
@@ -0,0 +1,44 @@
+
+### Synopsis
+
+Validates the syntax of a par2cron YAML configuration
+Use the command to check configurations before deploying
+
+Invalid configurations will prevent par2cron from starting;
+this command will exit with non-zero if the validation fails.
+
+Full documentation at: https://github.com/desertwitch/par2cron
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)